src/styles/global-styles.scss

```diff
@@ -167,3 +167,17 @@ button {
 img {
   vertical-align: middle;
 }
+
+/**
+ * Leaflet used high z-indexes by default, causing z-index issues, enforce low z-index for it
+ */
+.leaflet-control {
+  z-index: 0 !important;
```

Comment:
Using `!important` is not a best practice because it can be more complicated to figure out the cause of the wrong behavior.

I can offer 2 other ways to fix that:
1. Make the selector stronger by repeating it:
```css
.leaflet-control.leaflet-control {
  z-index: 0 !important;
}
```
2. Use cascade in Map component to make the selector stronger:

```css
.Map__container .leaflet-control {
  z-index: 0 !important;
}
```

Reply:
Great point, thanks, fixed: https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/pull/148/commits/ba18928932195402aae97bbdbcf5fb77752d0512